### PR TITLE
release-notes-2-8-0

### PR DESCRIPTION
**Pull request by purna-m**

### CODE REVIEW

**Comment by snneji** [comment]
Thanks @purna-m!

File: release-notes.html.md.erb [hunk 1]
```diff
@@ -41,10 +41,52 @@ changing server defaults, see [Changing MySQL Server Defaults](./change-default.
 utility to restore a HA cluster from a backup.
 For more information, see [Restoring <%= vars.product_short %>](./restore.html).
 
++ **Backup and restore workflow supported for Multi-site Leader-follower plans:** Multi-site replication instances will now be 
+backed up on the schedule configured by Operators. And Operators can use the `mysql-restore` utility to restore 
+a multi-site instance from a backup.
+
++ **Developer can now configure follower for replication from a non-empty multi-site instance in a different foundation.
```
Comment (aegershman) [inline release-notes.html.md.erb:48]:
```suggestion
+ **Developers can now configure follower for replication from a non-empty multi-site instance in a different foundation.
```

very minor was just reading through and saw pluralization of this

File: release-notes.html.md.erb [hunk 2]
```diff
@@ -41,10 +41,52 @@ changing server defaults, see [Changing MySQL Server Defaults](./change-default.
...
+
++ **Metrics for Multi-site Leader-follower plans:** Multi-site replication instances will now metrics which can be viewed in realtime
```
Comment (aegershman) [inline release-notes.html.md.erb:50]:
```suggestion
+ **Metrics for Multi-site Leader-follower plans:** Multi-site replication instances will now emit metrics which can be viewed in realtime
using `cf tail`.
```

File: release-notes.html.md.erb [hunk 3]
```diff
@@ -41,10 +41,52 @@ changing server defaults, see [Changing MySQL Server Defaults](./change-default.
...
+using `cf tail`.
+
++ An error message appears when developers bind apps to custom schemas with invalid names. The error message shows the valid 
+characters for custom schema names.
+
++ Operators can run the `recreate-all-service-instances` errand to re-create service instance VMs.
+
++ Application and service key bindings can start a connection successfully to service instances of single node, HA cluster,
```
Comment (aegershman) [inline release-notes.html.md.erb:58]:
```suggestion
+ Application and service key bindings can start a connection successfully to service instances of single node, HA cluster, and multi-site leader-follower plans due to less stringent BOSH DNS hostnames.
```

File: release-notes.html.md.erb [hunk 4]
```diff
@@ -41,10 +41,52 @@ changing server defaults, see [Changing MySQL Server Defaults](./change-default.
...
+
+### Resolved Issues
+
+This release includes the following fixes:
+
++ An inaccurate generic message no longer appears when you use `cf update-service` to migrate data between a highly available 
+(HA) cluster plan and a plan of another topology. MySQL for Pivotal Platform now fails fast to indicate 
+that this is an unsupported workflow.
+
++ The MySQL tile now configures the endpoint url for s3 backups, and does not always uses the default AWS endpoint.
```
Comment (aegershman) [inline release-notes.html.md.erb:83]:
```suggestion
+ The MySQL tile now configures the endpoint url for s3 backups, and does not always use the default AWS endpoint.
```

Comment (snneji) [reply release-notes.html.md.erb:83]: Hi @aegershman, Thank you for your suggestions. We tend to do a copyedit for release notes PRs after they have been merged. The release notes have been edited and they are live here: https://docs.pivotal.io/p-mysql/2-8/release-notes.html

Comment (aegershman) [reply release-notes.html.md.erb:83]: apologies about that, didn't mean to get in the way, thanks for all you all do 👍

Comment (snneji) [reply release-notes.html.md.erb:83]: No worries! 🙂

**Pull request merged**